pynumaflow/function/server.py

```diff
@@ -45,9 +46,15 @@ class UserDefinedFunctionServicer(udfunction_pb2_grpc.UserDefinedFunctionService
     >>> grpc_server.start()
     """
 
-    def __init__(self, map_handler: UDFMapCallable, sock_path=FUNCTION_SOCK_PATH):
+    def __init__(
+        self,
+        map_handler: UDFMapCallable,
+        sock_path=FUNCTION_SOCK_PATH,
+        max_message_size=MAX_MESSAGE_SIZE,
+    ):
         self.__map_handler: UDFMapCallable = map_handler
         self.sock_path = f"unix://{sock_path}"
+        self.max_message_size = max_message_size
```

Comment:
```suggestion
        self._max_message_size = max_message_size
```
Then let's add it as a property def.